### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by qkdreyer**
https://github.com/rgsystemes/kb/issues/7128

### CODE REVIEW

File: ext/doc/Cassandra.php [hunk 1]
```diff
@@ -200,204 +200,207 @@ final class Cassandra {
...
+    }
 
     /**
      * Creates a new ssl builder for constructing a SSLOptions object.
      *
      * @return \Cassandra\SSLOptions\Builder A SSL options builder with default settings
      */
-    public static function ssl() { }
-
+    public static function ssl(): \Cassandra\SSLOptions\Builder
```
Comment (Beny13) [inline ext/doc/Cassandra.php:403]: This is definitely not php5 compatible

File: ext/doc/Cassandra/BatchStatement.php [hunk 2]
```diff
@@ -56,6 +58,7 @@ public function __construct($type) { }
      *
      * @return \Cassandra\BatchStatement self
      */
-    public function add($statement, $arguments) { }
-
+    public function add($statement, $arguments = null): \Cassandra\BatchStatement
```
Comment (Beny13) [inline ext/doc/Cassandra/BatchStatement.php:61]: You could've used `: self` for quite a lot of these

File: ext/doc/Cassandra/Bigint.php [hunk 3]
```diff
@@ -21,103 +21,134 @@
 /**
  * A PHP representation of the CQL `bigint` datatype
  */
-final class Bigint implements Value, Numeric {
-
+final class Bigint implements \Stringable, Value, Numeric
```
Comment (Beny13) [inline ext/doc/Cassandra/Bigint.php:24]: Stringable is PHP8 only

File: ext/doc/Cassandra/BatchStatement.php (hunk 2 above)
Comment (qkdreyer) [reply ext/doc/Cassandra/BatchStatement.php:61]: I didn't manually edit these files, everything is generated from `generate_doc.php`

File: ext/doc/Cassandra/Bigint.php (hunk 3 above)
Comment (qkdreyer) [reply ext/doc/Cassandra/Bigint.php:24]: I didn't manually edit these files, everything is generated from `generate_doc.php`

File: ext/doc/Cassandra.php (hunk 1 above)
Comment (qkdreyer) [reply ext/doc/Cassandra.php:403]: I didn't manually edit these files, everything is generated from `generate_doc.php`

File: ext/doc/Cassandra/Bigint.php (hunk 3 above)
Comment (Beny13) [reply ext/doc/Cassandra/Bigint.php:24]: Yeah but you have to update composer.json

**Review by Beny13 — APPROVED** [review]

**Review by sandvige — APPROVED** [review]
Cant review all

**Pull request merged**